py/engdoc/contributing/index.md

```diff
@@ -0,0 +1,188 @@
+
+    For more information about how to get started with using
+    Firebase Genkit, please see: [User Guide](.)
+
+## Preparing your account
+
+### Create a GitHub account
+
+1. [Sign up](https://github.com/signup) for an account.
+
+2. Please [enable 2 factor authentication
+   (2FA)](https://docs.github.com/en/authentication/securing-your-account-with-two-factor-authentication-2fa)
+   after having created a GitHub account.
+
+    1.  Use the [Google Authenticator
+        app](https://support.google.com/accounts/answer/1066447?hl=en&co=GENIE.Platform%3DAndroid)
+        for your device.
+
+        === "Android"
+
+            [Google Authenticator app for Android](https://play.google.com/store/apps/details?id=com.google.android.apps.authenticator2&hl=en_US&pli=1)
+
+        === "iOS"
+
+            [Google Authenticator app for iOS](https://apps.apple.com/us/app/google-authenticator/id388497605)
+
+    2.  Use a physical security key such as the [Google
+        Titan](https://store.google.com/product/titan_security_key?hl=en-US).
+
+4. [Generate an SSH
+   key](https://docs.github.com/en/authentication/connecting-to-github-with-ssh/generating-a-new-ssh-key-and-adding-it-to-the-ssh-agent)
+   for your workstation to associate it with your GitHub account.
+
+    !!! tip
+
+        You can consider using a script such as the following to easily identify
+        your key.
+
+        ```bash
+        #!/usr/bin/env bash
+
+        UNAME_MRS=$(uname -mrs || echo 'unknown')
+        UNAME_OS="$(uname -s | tr '[:upper:]' '[:lower:]' || echo 'unknown')"
+        OS_TYPE="$(echo $OSTYPE | tr '[:upper:]' '[:lower:]' || echo 'unknown')"
+        CPU_ARCH="$(uname -m || echo 'unknown')"
+        OS_DISTRO="$(cat /etc/os-release 2>/dev/null | grep -E '^ID=.*' | sed -e 's/^ID=\(.*\)/\1/g')"
+
+        ssh-keygen -vvvv -t ed25519 -C "ssh://${USER}@$(hostname)/?arch=${CPU_ARCH}&os=${OS_TYPE}&distro=${OS_DISTRO}&timestamp=$(date +'%Y-%m-%dT%H:%M:%S')"
+        cat ~/.ssh/id_ed25519.pub >> "${HOME}/.ssh/authorized_keys"
+        cat ~/.ssh/id_ed25519.pub
+        ```
+
+        Save this to a file called `gen_ssh_key.sh` and run it as follows after
+        changing to its container directory:
+
+        ```bash
+        chmod a+x gen_ssh_key.sh
+        ./gen_ssh_key.sh
+        ```
+
+5. [Associate your SSH key with your GitHub account](https://docs.github.com/en/authentication/connecting-to-github-with-ssh/adding-a-new-ssh-key-to-your-github-account).
+
+6. If you're a Googler, also [associate your `@google.com` email
+   address](https://docs.github.com/en/account-and-profile/setting-up-and-managing-your-personal-account-on-github/managing-email-preferences/adding-an-email-address-to-your-github-account)
+   with your GitHub account and follow any other requirements to complete this
+   process (ask `pavelgj@`).
+
+### GitHub Organization Membership
+
+Please talk to `pavelgj@` to get yourself added to the appropriate groups for
```

Comment:
let's keep specific names out of the docs, just say "Genkit dev team (on discord)".